src/middleware/trace.rs

```diff
@@ -50,6 +50,7 @@ use crate::util::trace_attributes_from_request;
 #[derive(Default, Debug)]
 pub struct RequestTracing {
     route_formatter: Option<Rc<dyn RouteFormatter + 'static>>,
+    default_attributes: Option<Vec<KeyValue>>,
```

Comment:
this a good fit for the instrumentation scope, (i.e Tracer level attributes) which would be more efficient than adding it directly to each span.